Problem 209: explain what happens with parameters

kegor opened the pull request:
Initial iteration to fix #209.

Commit:
Problem 209: few cases of using Handler parameters weren't described yet.

Solution: was added a description of some work-cases which were not previously mentioned.

warpech commented:
@alemoi, as you are the author of this API, please review that this is correct.

warpech commented on docs/topic-guides/blendable-web-apps/view-model-attaching.md:
```diff
@@ -156,6 +156,24 @@ In the example above, converter passes handler parameters to token parameters, a
+Blender.MapUri("/op1/{?}", "token1");
+Blender.MapUri("/op2/{?}/{?}", "token1");
+
+Response resp = Self.GET("/op2/first/second");
```
Please add a comment in this line saying what will happen. If I understand correctly, it should be:

```csharp
Response resp = Self.GET("/op2/first/second"); //this will not call `/op1/{?}` because of a different number of parameters
```

alemoi replied:
Its simpler. Current URI matcher selects the most concrete URI handler possible, among all choices. Its not related to number of parameters. Obviously, for `/op2/first/second`, more concrete handler is `/op2/{?}/{?}` and not `/op1/{?}`

kegor replied:
Hi @alemoi, could you please take a look at my recently added changes (which uses your explanation)?

alemoi replied:
Yes, that's better. Approved!

Commit:
Merge remote-tracking branch 'origin/2.4' into 209-parameters-in-attached-uris

Commit:
Problem 209: the sentence about the example of the various number of parameters should be rewritten.

Solution: now the sentence says that URI matcher selecting more concrete URI handler.

kegor commented:
@warpech, could you please also take a small look when you'll find few mins for this?

warpech committed:
some text style fixes

warpech reviewed (approved):
I pushed some tiny improvements. If you agree, I think it can be merged
